pkg/dockerregistry/server/configuration/configuration.go

```diff
@@ -27,14 +88,21 @@ type openshiftConfig struct {
 }
 
 type Configuration struct {
-	Version  configuration.Version `yaml:"version"`
-	Metrics  Metrics               `yaml:"metrics"`
-	Requests Requests              `yaml:"requests"`
+	Version       configuration.Version
+	Metrics       Metrics
+	Requests      Requests
+	Server        *Server
+	Auth          *Auth
+	Audit         *Audit
+	Cache         *Cache
+	Quota         *Quota
+	Pullthrough   *Pullthrough
+	Compatibility *Compatibility
 }
 
 type Metrics struct {
-	Enabled bool   `yaml:"enabled"`
-	Secret  string `yaml:"secret"`
+	Enabled bool
+	Secret  string
```

Comment:
no, i'd like the yaml annotations put back in.  We should be consistent about providing them, even when it is a 1:1 mapping because it removes ambiguity and ensures the next person to add a field doesn't overlook doing it.